x/tokenfactory/keeper/admins_test.go

```diff
@@ -185,82 +185,76 @@ func (suite *KeeperTestSuite) TestBurnDenom() {
 }
 
 func (suite *KeeperTestSuite) TestChangeAdminDenom() {
-	var (
-		denom    string
-		addr0bal int64
-		addr1bal int64
-	)
-
 	for _, tc := range []struct {
-		desc  string
-		setup func()
-		valid bool
+		desc               string
+		msgChangeAdmin     func(denom string) *types.MsgChangeAdmin
+		expectedAdminIndex int
```

Comment:
Can we just make this `expectedAdmin` directly?

Reply:
You mean and then have them set as suite.TestAccs[0] and so on right?

Here we have the same problem where TestAccs are reseted to random accounts every iteration for each test, while the values for these fields are initialized at the very beginning

Reply:
Hrmm, not sure I get it, but not worth blocking this imo